build/Dockerfile

```
@@ -11,6 +11,8 @@ COPY build/_output/bin/integreatly-operator ${OPERATOR}
 # copy templates
 COPY templates /usr/local/bin/templates
 
+COPY products /usr/local/bin/products
```

```suggestion
COPY products /products
```
I think this is what's preventing prow tests to pass / run as prow seems to be using this dockerfile to build image :thinking:

---

Thanks @KevFan it was indeed related to the Docker container of the e2e tests. I used the same approach that is being used for the manifests folder, getting it from an env var and defaulting to the relative path if not set. Seems to be working now, hope the tests pass now :crossed_fingers:

---

Oh cool, that should work :+1: Though it may not be actually needed since we don't seem to be setting that env var for the release dockerfile anymore and have been installing fine :thinking: 
* https://github.com/integr8ly/integreatly-operator/blob/master/Dockerfile#L34

Maybe we should eventually revisit the differences between these 2 dockerfiles and decide whether the prow one is still needed / should be updated. Be good to keep them consistent enough or build from the release one :thinking: No harm in having the ability to set via env var though :+1:

---

Totally agree. If I remember correctly we ended up with these 2 Dockerfiles in order to unblock us when testing the operator-sdk bump in prow, but we should look into having only one Dockerfile